docs/dev_guide/diagnostic_ref/dpct1048.rst

```diff
@@ -22,4 +22,40 @@ or replaced with 0, depending on how they are used.
 Suggestions to Fix
 ------------------
 
-Review the code and adjust it.
\ No newline at end of file
+Review the code and adjust it.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo() {
+      double2 *h_A;
+      cudaHostAlloc(&h_A, sizeof(double2), cudaHostAllocDefault);
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done